lib/tasks/state_file_archive_intakes.rake

```diff
@@ -1,27 +1,38 @@
 namespace :state_file do
-  namespace :ty23 do
-    desc 'find archiveable az intakes and create their archival record'
-    task archive_az_intakes: :environment do
+  namespace :ty24 do
+    desc 'Archive intakes for a specific state'
+    task :archive, [:state_code] => :environment do |_t, args|
       Rails.logger = Logger.new($stdout)
-      batch_size = 10 # we batch these since archiving involves copying the submission pdf to a new location in s3
-      archiver = StateFile::Ty23ArchiverService.new(state_code: 'az', batch_size: batch_size)
-      archiver.find_archiveables # sets `current_batch` on the archiver instance
-      while archiver.current_batch.count.positive? # keep archiving, in batches, until the archiver doesn't find anything else
-        archiver.archive_batch # process the batch
-        archiver.find_archiveables # set the next batch
-      end
+      # we batch these since archiving involves copying the submission pdf to a new location in s3
+      StateFile::Ty24ArchiverService.archive!(
+        state_code: args[:state_code],
+        batch_size: 50
```

Comment:
last year the batch number was 10, it seemed a little stingy to me so I increased it but happy to go down again